azure-cron.yml

```diff
@@ -48,6 +48,28 @@ jobs:
           AWS_SECRET_ACCESS_KEY: $(AWS_SECRET_ACCESS_KEY)
       - template: ci/tell-slack-failed.yml
 
+  - job: fix_bazel_cache
+    timeoutInMinutes: 120
+    pool:
+      name: 'ubuntu_20_04'
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: |
+          set -euo pipefail
+          eval "$(dev-env/bin/dade assist)"
+
+          bazel build //ci/cron:cron
+          # 90 minutes should provide enough overlap for an hourly
+          # cronjob.
+          ./bazel-bin/ci/cron/cron bazel-cache --age 90 --delete
+        env:
+          AWS_ACCESS_KEY_ID: $(AWS_ACCESS_KEY_ID)
+          AWS_SECRET_ACCESS_KEY: $(AWS_SECRET_ACCESS_KEY)
```

Comment:
These are not the keys you're looking for.

You want:
```
        env:
          GCRED: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
```

and you also need to replicate the logic from [bash-lib](https://github.com/digital-asset/daml/blob/main/ci/bash-lib.yml#L74-L94), either in the Bash script here or in your Haskell code. (Or call Bash from Haskell, but some people find that messy.)

Reply:
Duh, I clearly should have taken a look at the credentials instead of blindly copying the docs cron. Thanks for catching it!